packages/shorebird_cli/lib/src/shorebird_android_artifacts.dart

```diff
@@ -0,0 +1,157 @@
+import 'package:path/path.dart' as p;
+import 'package:scoped/scoped.dart';
+import 'package:shorebird_cli/src/command.dart';
+import 'package:shorebird_cli/src/third_party/flutter_tools/lib/flutter_tools.dart';
```

Comment:
What is this import used for?

Reply:
apparently it exposes `dart:io` classes and I might auto selecting it, changed to `dart:io`